apps/site/assets/css/_footer.scss

```
@@ -9,47 +9,128 @@
   background-color: $footer-color;
   color: $white;
   min-width: 20rem;
-  padding-bottom: $base-spacing * 2;
-  padding-top: $base-spacing * 2;
+  padding-bottom: 24px;
+  padding-top: 24px;
+  border-radius:  0 50px 0 0;
+  flex-basis: 70%;
 
   @include media-breakpoint-up(xxl) {
     font-size: $font-size-base;
   }
 
-  @include media-breakpoint-down(md) {
+  @include media-breakpoint-up(lg) {
+    height: auto;
+    padding-left: 1rem;
+    position: relative;
+  }
+
+  @include media-breakpoint-only(md) {
     font-size: $font-size-base-sm;
+    z-index: 1;
+    position: absolute;
+    width: 100%;
   }
 
-  @media print {
-    display: none;
+  .container {
+    padding-bottom: 44px;
+    padding: 0 4px 0 4px;
+
+    @include media-breakpoint-up(lg) {
+      padding-left: 0px;
+      padding-right: 4px;
+    }
+
+
   }
 }
 
-.m-footer__group {
-  @include media-breakpoint-up(md) {
-    @include make-col();
-    @include make-col-span(3);
-    padding-bottom: $base-spacing * 2;
+.m-footer__links {
+  display: grid;
+  gap: 1.75rem;
+  grid-template-columns: 1fr 1fr 1fr;
+
+  .m-menu__feature {
```

```suggestion
  .m-menu__feature {
    width: initial;
```
To unset the `width: calc(100% - 40px);` that's currently set on it. This way it'll take up the full width available, filling out some of the extra space that's currently to the right.


https://user-images.githubusercontent.com/2136286/189409672-49fe0527-916d-4169-880d-0ee1a502f271.mov